cfgov/ask_cfpb/scripts/export_ask_data.py

```diff
@@ -0,0 +1,143 @@
+
+import HTMLParser
+from django.utils import html
+
+from paying_for_college.csvkit import csvkit
+
+from ask_cfpb.models import Answer
+
+html_parser = HTMLParser.HTMLParser()
+
+HEADINGS = [
+    'ASK_ID',
+    'Question',
+    'ShortAnswer',
+    'Answer',
+    'URL',
+    'SpanishURL',
+    'Topic',
+    'SubCategories',
+    'Audiences',
+    'RelatedQuestions',
+    'RelatedResources',
+]
+
+SPANISH_HEADINGS = [
+    'ASK_ID'.encode('UTF16'),
+    'SpanishURL'.encode('UTF16'),
+    'SpanishAnswer'.encode('UTF16'),
+]
+
+
+def clean_and_strip(data):
+    unescaped = html_parser.unescape(data)
+    return html.strip_tags(unescaped).strip()
+
+
+def clean_and_strip_spanish(data):
+    unescaped = html_parser.unescape(data)
+    return html.strip_tags(
+        unescaped).strip().replace(
+        '\t', ' ').replace(
+        '\n', ' ').replace(
+        '\r', ' ')
+
+
+def assemble_output(spanish_only=False):
+    answers = Answer.objects.all()
+    output_rows = []
+    if spanish_only:
+        for answer in answers:
+            output = {
+                'ASK_ID'.encode('UTF16'):
+                str(answer.id).encode('UTF16'),
+                'SpanishURL'.encode('UTF16'): (
+                    answer.spanish_page.url_path.replace(
+                        '/cfgov', '').encode('UTF16')
+                    if answer.spanish_page else ''.encode('UTF16')),
+                'SpanishAnswer'.encode('UTF16'): clean_and_strip_spanish(
+                    answer.answer_es).encode('UTF16')
+            }
+            output_rows.append(output)
+        return output_rows
+    for answer in answers:
+        output = {heading: '' for heading in HEADINGS}
+        output['ASK_ID'] = answer.id
+        output['Question'] = answer.question
+        output['ShortAnswer'] = clean_and_strip(
+            answer.snippet)
+        output['Answer'] = clean_and_strip(
+            answer.answer)
+        # output['SpanishAnswer'] = clean_and_strip(
+        #     answer.answer_es)
+        output['URL'] = answer.english_page.url_path.replace(
+            '/cfgov', '') if answer.english_page else ''
+        output['SpanishURL'] = (
+            answer.spanish_page.url_path.replace(
+                '/cfgov', '') if answer.spanish_page else '')
+        output['Topic'] = answer.category.first().name
+        output['SubCategories'] = " | ".join(
+            [subcat.name for subcat in answer.subcategory.all()])
+        output['Audiences'] = " | ".join(
+            aud.name for aud in answer.audiences.all())
+        output['RelatedQuestions'] = " | ".join(
+            ['{}'.format(a.id) for a in answer.related_questions.all()])
+        output['RelatedResources'] = (
+            answer.next_step.title
+            if answer.next_step
+            else '')
+        output_rows.append(output)
+    return output_rows
+
+
+def export_questions(spanish_only=False):
+    """
+    A rather ridiculous script for exporting Ask CFPB Answer content.
```

Comment:
I hope the friendliness of the Wagtail interface will make these dump requests obsolete!